pr_check.sh

```diff
@@ -31,6 +31,13 @@ IMAGES=""
 export COMPONENT_NAME="rhsm"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
 # prebuild artifacts for quarkus builds
 for service in $SERVICES; do
+  # Ignore the swatch-metrics service until it's fully ported.
+  # To be uncommented in SWATCH-1806.
+  if [ "$service" == "swatch-metrics" ]
+  then
+    continue;
```

Comment:
Just a comment for more context about what ignoring here is actually doing.

This is basically doing two things:

1. Skipping building the new `:swatch-metrics` project (and also not pushing it up to quay.io/cloudservices/swatch-metrics)
2. Preventing `-i quay.io/cloudservices/swatch-metrics=$PRTAG` from being included in the bonfire deploy command

What this ultimately means is that when pr_check.sh gets ran and an ephemeral environment deploys, the swatch-metrics and swatch-metrics-rhel pods are going to be running code from the monolith (bundled in the quay.io/cloudservices/rhsm-subscriptions image) still.